plugins/woocommerce/src/Admin/Features/Settings.php

```diff
@@ -61,22 +61,44 @@ public function enqueue_settings_editor_scripts() {
 		// Make sure the Settings Editor package is loaded.
 		wp_enqueue_script( 'wc-settings-editor' );
 
-		$script_handle          = 'wc-admin-edit-settings';
-		$script_path_name       = 'wp-admin-scripts/settings';
-		$script_assets_filename = WCAdminAssets::get_script_asset_filename( 'wp-admin-scripts', 'settings' );
-		$script_assets          = require WC_ADMIN_ABSPATH . WC_ADMIN_DIST_JS_FOLDER . 'wp-admin-scripts/' . $script_assets_filename;
-		$script_version         = WCAdminAssets::get_file_version( 'js', $script_assets['version'] );
-
-		wp_register_script(
-			$script_handle,
-			WCAdminAssets::get_url( $script_path_name, 'js' ),
-			$script_assets['dependencies'],
-			$script_version,
+		$script_name            = 'wc-admin-edit-settings';
+		$script_path_name       = 'settings';
+		$script_assets_filename = WCAdminAssets::get_script_asset_filename( $script_path_name, 'index' );
+		$script_assets          = require WC_ADMIN_ABSPATH . WC_ADMIN_DIST_JS_FOLDER . $script_path_name . '/' . $script_assets_filename;
+		$style_assets_filename  = WCAdminAssets::get_script_asset_filename( $script_path_name, 'style' );
+		$style_assets           = require WC_ADMIN_ABSPATH . WC_ADMIN_DIST_JS_FOLDER . $script_path_name . '/' . $style_assets_filename;
+
+		wp_enqueue_script(
```

Comment:
Should we also call [`wp_set_script_translations`](https://developer.wordpress.org/reference/functions/wp_set_script_translations/)? I think this is needed when we have a script that is localized and we want to load the translations for it.

Reply:
Indeed, yes. Done in https://github.com/woocommerce/woocommerce/pull/52501/commits/b35c56cc2ed0e5a9eaf654f7f9d82b250e8a94dc